extensions/amp-timeago/1.0/timeago.js

```diff
@@ -49,25 +53,29 @@ export function Timeago(props) {
   }, [datetime, locale, cutoff, cutoffText]);
 
   useResourcesNotify();
-  return createElement('time', {datetime, ref}, timestamp);
+  return (
+    <time datetime={datetime} ref={ref} {...rest}>
+      {timestamp}
+    </time>
+  );
 }
 
 /**
  * @param {string} datetime
  * @param {string} locale
- * @param {string} cutoffText
  * @param {number=} opt_cutoff
+ * @param {string=} opt_cutoffText
  * @return {string}
  */
-function getFuzzyTimestampValue(datetime, locale, cutoffText, opt_cutoff) {
+function getFuzzyTimestampValue(datetime, locale, opt_cutoff, opt_cutoffText) {
```

Comment:
These do not seem to be optional (`=`) args - they are always passed from what I can tell. They are just `|undefined` and no need for "opt_" prefix.